ctrd/container.go

```diff
@@ -14,11 +15,15 @@ import (
 
 	"github.com/containerd/containerd"
 	containerdtypes "github.com/containerd/containerd/api/types"
+	"github.com/containerd/containerd/archive"
+	"github.com/containerd/containerd/content"
 	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/images"
 	"github.com/containerd/containerd/leases"
 	"github.com/containerd/containerd/linux/runctypes"
 	"github.com/containerd/containerd/oci"
 	"github.com/docker/docker/pkg/stdcopy"
+	"github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
could we use `imagespec` instead of `v1` ? `v1` doesn't contain any useful information.